tests/test_misc.py

```diff
@@ -123,3 +124,19 @@ def test_throttle_event_types():
     assert isinstance(throttle_event.broker_id, int) and throttle_event.broker_id == 0
     assert isinstance(throttle_event.throttle_time, float) and throttle_event.throttle_time == 10.0
     assert str(throttle_event) == "broker/0 throttled for 10000 ms"
+
+
+@pytest.mark.skipif(len([True for x in (".so", ".dylib", ".dll")
+                         if os.path.exists("monitoring-interceptor" + x)]) == 0,
+                    reason="requires confluent-librdkafka-plugins be installed and copied to the current directory")
+def test_unordered_dict():
+    """
+    Interceptor configs can only be handled after the plugin has been loaded not before.
+    """
+    for fn in [confluent_kafka.Producer, confluent_kafka.Consumer, confluent_kafka.admin.AdminClient]:
```

Comment:
I was thinking you could use parameterization: https://docs.pytest.org/en/latest/parametrize.html